packages/stream/solana/constants.ts

```diff
@@ -26,6 +26,19 @@ export const PROGRAM_ID = {
   [ICluster.Local]: "HqDGZjaVRXJ9MGRQEw7qDc2rAr6iH1n1kAQdCZaCMfMZ",
 };
 
+export const ALIGNED_UNLOCKS_PROGRAM_ID = {
+  [ICluster.Devnet]: "aSTRM2NKoKxNnkmLWk9sz3k74gKBk9t7bpPrTGxMszH",
+  [ICluster.Mainnet]: "aSTRM2NKoKxNnkmLWk9sz3k74gKBk9t7bpPrTGxMszH",
+  [ICluster.Testnet]: "aSTRM2NKoKxNnkmLWk9sz3k74gKBk9t7bpPrTGxMszH",
+  [ICluster.Local]: "aSTRM2NKoKxNnkmLWk9sz3k74gKBk9t7bpPrTGxMszH",
```

Comment:
@Yolley do we need this, right now I don't use it but I am guessing we can override the `programId` in the constructor of the `Program` in case we use different addresses for devnet/mainnet

Reply:
It used to derive PDAs. If a consumer of the sdk wants to get account addresses, these constants must be public, right?